spec/factories/notifications.rb

```diff
@@ -1,26 +1,63 @@
+# frozen_string_literal: true
+
 FactoryGirl.define do
-  factory :apns_notification, class: 'RubyPushNotifications::APNS::APNSNotification' do
+  factory :apns_notification,
+          class: 'RubyPushNotifications::APNS::APNSNotification' do
     tokens { [generate(:apns_token)] }
     data a: 1
 
     initialize_with { new tokens, data }
   end
 
-  factory :gcm_notification, class: 'RubyPushNotifications::GCM::GCMNotification' do
+  factory :gcm_notification,
+          class: 'RubyPushNotifications::GCM::GCMNotification' do
     registration_ids { [generate(:gcm_registration_id)] }
     data a: 1
 
     initialize_with { new registration_ids, data }
   end
 
-  factory :mpns_notification, class: 'RubyPushNotifications::MPNS::MPNSNotification' do
+  factory :fcm_notification,
+          class: 'RubyPushNotifications::FCM::FCMNotification' do
+    registration_ids { [generate(:fcm_registration_id)] }
+    data a:
+           {
+             'notification': {
```

Comment:
It looks like older versions or ruby don't accept this combination of strings as keys and colons as separators. They require rockets instead (`=>`). Would you mind changing them so that the travis CI build passes? (See https://travis-ci.org/calonso/ruby-push-notifications/jobs/565406030)

Reply:
There we go, that was actually a paste from my original code that we use to push to devices from our live system. the payload was adjusted according to that script so the changes was actually supposed to be done in the first place :smile: it's corrected and the build is passing as you can see now.